### PR TITLE
Updates on subgraph matching for graph challenge tests

### PR DESCRIPTION
**Pull request by Laurawly**
@neoblizz Could you take a look?

### CODE REVIEW

File: examples/sm/run.sh [hunk 1]
```diff
@@ -1 +1 @@
-./bin/sm_main_9.0_x86_64 market  ../../dataset/small/tree.mtx --pattern-graph-type=market --pattern-graph-file=../../dataset/small/query_sm.mtx --undirected=1 --pattern-undirected=1 --num-runs=1 --quiet=False
+./bin/sm_main_10.0_x86_64 market  ../../dataset/small/tree.mtx --pattern-graph-type=market --pattern-graph-file=../../dataset/small/query_sm.mtx --undirected=1 --pattern-undirected=1 --num-runs=1 --quiet=False
```
Comment (neoblizz) [inline examples/sm/run.sh]: Can you add more than just one small example in the `run.sh` file? And use other examples to make sure we follow some sort of standard for this, for example, you hard code the binary name (which will change depending on what CUDA version you use) and some of the other files actually search for the latest binaries.

**Comment by neoblizz** [comment]
@Laurawly `    /var/lib/jenkins/workspace/gunrock_PR-582/examples/tc/test_tc.cu` can you rename it back to `test_tc.cu`?

**Comment by Laurawly** [comment]
> @Laurawly ` /var/lib/jenkins/workspace/gunrock_PR-582/examples/tc/test_tc.cu` can you rename it back to `test_tc.cu`?

Sure, I'll do it.

File: examples/sm/run.sh (hunk 1 above)
Comment (Laurawly) [reply examples/sm/run.sh]: Sure, that makes sense.

**Comment by Laurawly** [comment]
> This commit doesn't add `tc_test.cu` back in after deleting `tc_main.cu`.

Forgot to add it back. Committed.

**Comment by neoblizz** [comment]
@Laurawly this is the output from jenkins:
```
26: Test command: /var/lib/jenkins/workspace/gunrock_PR-582/build/bin/sm "market" "/var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/tree.mtx" "--pattern-graph-type=market" "--pattern-graph-file=/var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/query_sm.mtx" "--undirected=1" "--pattern-undirected=1" "--num-runs=1" "--quiet=false"

26: Test timeout computed to be: 9.99988e+06

26: Loading Matrix-market coordinate-formatted graph ...

26:   Reading from /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/tree.mtx:

26:   Parsing MARKET COO format (5 nodes, 5 directed edges)...   Done (0 s).

26:   Writing meta data into /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/tree.mtx.meta

26:   Writting edge pairs in binary into /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/tree.mtx.coo_edge_pairs

26:   Assigning 1 to all 5 edges

26:   Substracting 1 from node Ids...

26:   Edge doubleing: 5 -> 10 edges

26:   graph loaded as COO in 0.032215s.

26: Converting 5 vertices, 10 directed edges ( ordered tuples) to CSR format...Done (0s).

26: Degree Histogram (5 vertices, 10 edges):

26:     Degree 0: 0 (0.000000 %)

26:     Degree 2^0: 2 (40.000000 %)

26:     Degree 2^1: 2 (40.000000 %)

26:     Degree 2^2: 1 (20.000000 %)

26: 

26: Loading Matrix-market coordinate-formatted pattern-graph ...

26:   Reading from /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/query_sm.mtx:

26:   Parsing MARKET COO format (3 nodes, 3 directed edges)...   Done (0 s).

26:   Writing meta data into /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/query_sm.mtx.meta

26:   Writting edge pairs in binary into /var/lib/jenkins/workspace/gunrock_PR-582/dataset/small/query_sm.mtx.coo_edge_pairs

26:   Assigning 1 to all 3 edges

26:   Substracting 1 from node Ids...

26:   Edge doubleing: 3 -> 6 edges

26:   pattern-graph loaded as COO in 0.029946s.

26: Converting 3 vertices, 6 directed edges ( ordered tuples) to CSR format...Done (0s).

26: Degree Histogram (3 vertices, 6 edges):

26:     Degree 0: 0 (0.000000 %)

26:     Degree 2^0: 0 (0.000000 %)

26:     Degree 2^1: 3 (100.000000 %)

26: 

26: # of nodes in data graph: 5

26: # of nodes in query graph: 3

26: __________________________

26: CPU_Reference: start

26: CPU_Reference: done

26: __________________________

26: Run CPU Reference Avg. in 1 iterations elapsed: 0.002146 ms

26: ==============================================

26: 64bit-VertexT=false 64bit-SizeT=false 64bit-ValueT=true undirected=true advance-mode=LB

26: subgraph nodes: 5, query graph nodes:3

26: Using advance mode LB

26: Using filter mode CULL

26: __________________________

26: counter :  |(0) 0

26: counter :  |(0) 0

26: counter :  |(0) 1

26: --------------------------

26: Run 0 elapsed: 40.143013 ms, #iterations = 1

26: Validate_Results

26: Subgraph Matching Validity: FAIL: [0]: 0 != 1

26: result[...0, 1, 0, 1, 0, ...]

26: reference[...1, 1, 0, 1, 0, ...]

26: 1 errors occurred.

26: [SM] finished.

26:  avg. elapsed: 40.143013 ms

26:  iterations: 1

26:  min. elapsed: 40.143013 ms

26:  max. elapsed: 40.143013 ms

26:  load time: 100.514 ms

26:  preprocess time: 322.853000 ms

26:  postprocess time: 0.270844 ms

26:  total time: 380.952835 ms

26/34 Test #26: TEST_SM ..........................***Failed  Required regular expression not found.Regex=[PASS

]  0.53 sec

test 27

      Start 27: TEST_SS_CMD
```

**Comment by Laurawly** [comment]
@neoblizz Could you try it again?

**Pull request merged**